app/components/homeScreen/DailyReads.tsx

```diff
@@ -60,8 +58,13 @@ const DailyReads = ():any => {
     (state: any) =>
       state.utilsData.ActivitiesData != '' ? JSON.parse(state.utilsData.ActivitiesData) : [],
   );
+  const childAge = useAppSelector(
+    (state: any) =>
+      state.utilsData.taxonomy.allTaxonomyData != '' ? JSON.parse(state.utilsData.taxonomy.allTaxonomyData).child_age : [],
+  );

```

Comment:
You can add try catch block to handle JSON parsing errrors.

```
const childAge = useAppSelector((state: RootState) => {
  const allTaxonomyData = state.utilsData.taxonomy.allTaxonomyData;

  if (!allTaxonomyData) {
    return [];
  }

  try {
    const parsedData = JSON.parse(allTaxonomyData);
    return parsedData.child_age || [];
  } catch (error) {
    console.error("Failed to parse allTaxonomyData:", error);
    return [];
  }
});
```

Reply:
okay we can use this try catch